scripts/tacc_examples/SAGE_2021_script.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#SBATCH -J mspass          # Job name
+#SBATCH -o mspass.o%j      # Name of stdout output file
+#SBATCH -p normal          # Queue (partition) name
+#SBATCH -N 2               # Total # of nodes (must be 1 for serial)
+#SBATCH -n 2               # Total # of mpi tasks (should be 1 for serial)
+#SBATCH -t 02:00:00        # Run time (hh:mm:ss)
+#SBATCH -A MsPASS          # Allocation name (req'd if you have more than 1)
+
+# working directory
+WORK_DIR=$SCRATCH/SAGE_2021
+# directory where contains docker image
+MSPASS_CONTAINER=$WORK2/mspass/mspass_latest.sif
```

Comment:
I think the work filesystem is already migrated. We should just use $WORK instead of $WORK2